sonar/requirements.sh

```
@@ -0,0 +1,4 @@
+wget https://services.gradle.org/distributions/gradle-6.9.4-bin.zip -P /tmp
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)